<?xml version="1.0" encoding="UTF-8"?>
<results version="2">
    <cppcheck version="2.16.0"/>
    <errors>
        <error id="normalCheckLevelMaxBranches" severity="information" msg="Limiting analysis of branches. Use --check-level=exhaustive to analyze all branches." verbose="Limiting analysis of branches. Use --check-level=exhaustive to analyze all branches." file0="F:/Benchmark/GitMaterial_tongyi/ProblemInfoCrawler/output/c/Hard/2111-the-number-of-good-subsets.c">
            <location file="F:\Benchmark\GitMaterial_tongyi\ProblemInfoCrawler\output\c\Hard\2111-the-number-of-good-subsets.c" line="0" column="0"/>
        </error>
        <error id="arrayIndexOutOfBounds" severity="error" msg="Array &apos;prime[30]&apos; accessed at index 30, which is out of bounds." verbose="Array &apos;prime[30]&apos; accessed at index 30, which is out of bounds." cwe="788" file0="F:/Benchmark/GitMaterial_tongyi/ProblemInfoCrawler/output/c/Hard/2111-the-number-of-good-subsets.c">
            <location file="F:\Benchmark\GitMaterial_tongyi\ProblemInfoCrawler\output\c\Hard\2111-the-number-of-good-subsets.c" line="41" column="18" info="Array index out of bounds"/>
            <location file="F:\Benchmark\GitMaterial_tongyi\ProblemInfoCrawler\output\c\Hard\2111-the-number-of-good-subsets.c" line="40" column="19" info="Assuming that condition &apos;i&lt;=30&apos; is not redundant"/>
        </error>
        <error id="arrayIndexOutOfBounds" severity="error" msg="Array &apos;cnt[30]&apos; accessed at index 30, which is out of bounds." verbose="Array &apos;cnt[30]&apos; accessed at index 30, which is out of bounds." cwe="788" file0="F:/Benchmark/GitMaterial_tongyi/ProblemInfoCrawler/output/c/Hard/2111-the-number-of-good-subsets.c">
            <location file="F:\Benchmark\GitMaterial_tongyi\ProblemInfoCrawler\output\c\Hard\2111-the-number-of-good-subsets.c" line="46" column="16" info="Array index out of bounds"/>
            <location file="F:\Benchmark\GitMaterial_tongyi\ProblemInfoCrawler\output\c\Hard\2111-the-number-of-good-subsets.c" line="45" column="19" info="Assuming that condition &apos;i&lt;=30&apos; is not redundant"/>
        </error>
        <error id="constParameterPointer" severity="style" msg="Parameter &apos;nums&apos; can be declared as pointer to const" verbose="Parameter &apos;nums&apos; can be declared as pointer to const" cwe="398" file0="F:/Benchmark/GitMaterial_tongyi/ProblemInfoCrawler/output/c/Hard/2111-the-number-of-good-subsets.c">
            <location file="F:\Benchmark\GitMaterial_tongyi\ProblemInfoCrawler\output\c\Hard\2111-the-number-of-good-subsets.c" line="35" column="30" info="Parameter &apos;nums&apos; can be declared as pointer to const"/>
            <symbol>nums</symbol>
        </error>
        <error id="unusedFunction" severity="style" msg="The function &apos;numberOfGoodSubsets&apos; is never used." verbose="The function &apos;numberOfGoodSubsets&apos; is never used." cwe="561">
            <location file="F:\Benchmark\GitMaterial_tongyi\ProblemInfoCrawler\output\c\Hard\2111-the-number-of-good-subsets.c" line="35" column="0"/>
            <symbol>numberOfGoodSubsets</symbol>
        </error>
        <error id="checkersReport" severity="information" msg="Active checkers: 108/836 (use --checkers-report=&lt;filename&gt; to see details)" verbose="Active checkers: 108/836 (use --checkers-report=&lt;filename&gt; to see details)"/>
    </errors>
</results>
